### ISSUE TITLE
Synchronising css classes with `ts.draw_svg()`

### ISSUE THREAD

**Issue by hyanwong** [kitchensjn/tskit_arg_visualizer#18]
I see that nodes have css classes in the d3js viz, and therefore presumably can be styled individually. 

I wonder if it is worth figuring out if there is any way to align the terminology with the css classes described at https://tskit.dev/tutorials/viz.html#styling? The problem is that the tree-base visualizations have a nested structure that I think doesn't make sense for a graph-based viz, so it's not as simple as trying to make a `draw_svg()` stylesheet work seamlessly in the ARG visualizer.

FWIW, node symbols are labelled with the `sim` class, node labels with the `lab` class, and they are grouped into a higher-level `node` class, which has additional classes. The example given in the docs is a node grouping with:

```
<g class="a10 i3 leaf m16 m17 node n7 p2 s15 s16 sample">
```

with the description:

> The classes indicate that this is a sample node with an immediate ancestor (parent) node of ID 10 (a10), and that the node belongs to an [individual](https://tskit.dev/tskit/docs/stable/glossary.html#sec-nodes-or-individuals) with ID 3 (i3). The classes n7 and p2 tell us that the node ID is 7 and is is from the population with ID 2 (p2). Other ID classes on the node tell us about the mutations above that node, of which there are two in this case, with IDs 16 and 17 (m16, m17); those mutations are associated with site IDs 15 and 16 (s16, s17).

**Comment by kitchensjn** [comment]
I think that's a great idea! Will require some reworking to get it structured in a similar way. I will look into this once I'm back from traveling (currently out of the country).

**Comment by kitchensjn** [comment]
I need to be a bit more careful about how I go about synchronizing, as we may want different styles for different figures. For example, the scoping of the visualizer CSS is too broad which is affecting other figures on the tutorials document:

<img width="594" alt="Screenshot 2023-11-21 at 12 29 44 PM" src="https://github.com/kitchensjn/tskit_arg_visualizer/assets/40303683/c6a2ee94-db97-4417-a85a-fb6093bd79a8">

I'll plan to fix this particular styling in v0.0.2, but not going to add this to that milestone because I haven't really made much progress on this issue overall.

**Comment by hyanwong** [comment]
Yeah, it's a pain that stylesheets are not contained within a HTML doc. The normal SVGs have most of their content within a `.tree-sequence` class, so I guess you could have a similar `.arg` class.

Perhaps better would be to attach a class to the `<svg>` tag itself, e.g. `tskit-tree-plot` for the normal visualisations, and `tskit-arg-plot` for the canvas that contains your D3 plots?

**Comment by hyanwong** [comment]
Just to note that each argviz svg canvas now has a unique ID of the form `#arg_12345`, so it should be relatively easy to target specific canvasses with styles. However, it might be good to return the UID from each `draw` command by default, so it can be used programatically?
